src/router.js

```diff
@@ -33,5 +38,46 @@ export default [
 	{ path: '/tv/:id', name: 'tvShowDetails', component: TvShowDetails },
 	{ path: '/tv/:id/seasons', name: 'seasons', component: Seasons },
 	{ path: '/tv/:id/seasons/:number', name: 'episodes', component: Episodes },
-	{ path: '/person/:id', name: 'actorDetails', component: ActorDetails }
+	{ path: '/person/:id', name: 'actorDetails', component: ActorDetails },
+	{
+		path: '/login',
+		name: 'login',
+		component: Login,
+		meta: {
+			requiresAuth: true
```

Comment:
Login does not `requiresAuth` it requires that user is not authenticated. Rename it to something like `requiresNonAuthenticatedState`.